Modules/System/Cryptography Management/src/X509Certificate2.Codeunit.al

```diff
@@ -0,0 +1,104 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+/// <summary>
+/// Provides helper functions to work with X509Certificate2 class
+/// </summary>
+codeunit 1286 "X509Certificate2"
+{
+    Access = Public;
+
+    var
+        X509Certificate2Impl: Codeunit "X509Certificate2 Impl.";
+
+        /// <summary>
+        /// Verify that Certificate is initialized and exportable
+        /// </summary>
+        /// <param name="CertBase64Value">Certificate Base64 value</param>
+        /// <param name="Password">Certificate Password</param>
+        /// <param name="ContentType">Specifies the format of an X.509 certificate</param>
+        /// <returns>True if Certificate is verified</returns>
+    procedure VerifyCertificate(var CertBase64Value: Text; Password: Text; ContentType: Enum "X509 Content Type"): Boolean
```

Comment:
We have coding guidelines where one of the rules is that the variable name must match name of the object. Can you please change ContentType variable to match Enum name?